From bba20934da0ab4066f178ab278c0728b3ef49d16 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 22 Aug 2022 16:42:16 +0000
Subject: [PATCH] libstdc++: fix incomplete type of ui::LinuxInputMethodContext

Destructor of std::unique_ptr in libstdc++ uses sizeof() which
requires full definition of ui::LinuxInputMethodContext for return
type of gtk::GtkUiPlatformStub::CreateInputMethodContext().

Bug: 957519
Change-Id: I0e74265e1f352afec374dc6b0abb6d6b79371cf1
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3845078
Reviewed-by: Thomas Anderson <thomasanderson@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/main@{#1037800}
---
 ui/gtk/gtk_ui_platform_stub.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/gtk/gtk_ui_platform_stub.cc b/ui/gtk/gtk_ui_platform_stub.cc
index 5f01c8bd8ffda..bfd10bc796109 100644
--- a/ui/gtk/gtk_ui_platform_stub.cc
+++ b/ui/gtk/gtk_ui_platform_stub.cc
@@ -5,6 +5,7 @@
 #include "ui/gtk/gtk_ui_platform_stub.h"
 
 #include "base/callback.h"
+#include "ui/base/ime/linux/linux_input_method_context.h"
 
 namespace gtk {
 
